### PR TITLE
WIP: RHEL/Centos Support

### PR DESCRIPTION
**Pull request by jt6211**

### CODE REVIEW

File: scripts/install_mhnserver.sh [hunk 1]
```diff
@@ -2,19 +2,59 @@
...
+    PYTHON=/usr/local/bin/python2.7
+    PIP=/usr/local/bin/pip2.7
+    $PIP install virtualenv
+    VIRTUALENV=/usr/local/bin/virtualenv
+
+    #install supervisor from pip2.7
+    $PIP install supervisor
+    
+else
+    echo "ERROR: Unknown OS\nExiting!"
```
Comment (jt6211) [inline scripts/install_mhnserver.sh]: should be:
```
echo -e 'ERROR: Unknown OS\nExiting!'
```

File: scripts/install_mnemosyne.sh [hunk 2]
```diff
@@ -3,17 +3,49 @@
...
+        $SCRIPTDIR/install_python2.7.sh
+    fi
+
+    #use python2.7
+    PYTHON=/usr/local/bin/python2.7
+    PIP=/usr/local/bin/pip2.7
+    VIRTUALENV=/usr/local/bin/virtualenv
+
+else
+    echo "ERROR: Unknown OS\nExiting!"
```
Comment (jt6211) [inline scripts/install_mnemosyne.sh]: should be:
```
echo -e 'ERROR: Unknown OS\nExiting!'
```

File: server/requirements.txt [hunk 3]
```diff
@@ -1,12 +1,12 @@
 argparse==1.2.1
-Flask==0.10.1
-Flask-Login==0.2.11
-Flask-Mail==0.9.1
-Flask-Migrate==1.2.0
-Flask-SQLAlchemy==2.0
-Flask-Security==1.7.4
-Flask-Testing==0.4.2
-Flask-WTF==0.10.3
+Flask>=0.10.1
```
Comment (jt6211) [inline server/requirements.txt]: we've had trouble in the past with incompatible libs due to flexible versions like this (mainly with Flask-Security).  i recommend reverting or pinning to the new exact versions (after testing that they work together).

Comment (mwollenweber) [inline server/requirements.txt]: I can revert these changes. I had an issue with flask-sqlalchemy->sqlite that i thought was from pinned versions being incompatible. The actual issue was multiple version of sqlite-devel and has now been fixed in the install.

**Comment by jt6211** [comment]
tested on Digtial Ocean Centos 6.7.

**Pull request merged**